Makefile.in

```diff
@@ -0,0 +1,33 @@
+BINDIR = @BINDIR@
+LIBDIR = @LIBDIR@
+SYSCONFDIR = @SYSCONFDIR@
+MANDIR = @MANDIR@
+DATADIR = @DATADIR@
+NAME = @NAME@
+VERSION = @VERSION@
+
+.PHONY: install
+install:
```

Comment:
We probably want a note somewhere for Perl developers explaining why we're not using ExtUtils::MakeMaker or Module::Build. Maybe it even makes sense to have a no-op Makefile.PL`?

Reply:
I've thought about this more, and a Makefile.PL is a bad idea. It would make this look like a CPAN distribution, and we're explicitly not trying to organize this project that way.